### PR TITLE
Player

### PR DESCRIPTION
**Pull request by ChristopherKallstrom**

### CODE REVIEW

File: GitWorkflow/GitWorkflow/Player.hpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	Player(const sf::Vector2f& size = sf::Vector2f(100.f, 100.f), const sf::Color& color = sf::Color::Red)
+		: m_shape(size)
+	{
+		m_shape.setFillColor(color);
+	}
+
+	~Player() = default;
+
+private:
+	sf::RectangleShape m_shape;
```
Comment (Smaugmuel) [inline GitWorkflow/GitWorkflow/Player.hpp]: More descriptive name like "model" or similar

File: GitWorkflow/GitWorkflow/Player.hpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <SFML/Graphics.hpp>
+
+class Player final : public sf::Drawable
```
Comment (Smaugmuel) [inline GitWorkflow/GitWorkflow/Player.hpp:5]: For the future, we might refactor Drawables. But for now, it's fine to do it like this.

**Review by Smaugmuel — CHANGES_REQUESTED** [review]
Player.hpp is not included in the project. Include it, and make sure the project file is updated.

File: GitWorkflow/GitWorkflow/Player.hpp (hunk 1 above)
Comment (ChristopherKallstrom) [reply GitWorkflow/GitWorkflow/Player.hpp]: Included Player.hpp to project

File: GitWorkflow/GitWorkflow/Player.hpp (hunk 2 above)
Comment (ChristopherKallstrom) [reply GitWorkflow/GitWorkflow/Player.hpp:5]: Will refactor if needed in the future.

**Review by Smaugmuel — APPROVED** [review]
Looks good

**Pull request merged**